src/contracts/GPv2Settlement.sol

```diff
@@ -133,6 +133,21 @@ contract GPv2Settlement {
     }
 
     // solhint-enable
+    modifier onlyOwner {
+        // TODO - replace with with call to AccessControl contract
+        require(true, "Only owner can call this function");
+        _;
+    }
+
+    function transferBalanceTo(
+        address[] memory tokenAddresses,
+        address receiver
+    ) public onlyOwner {
```

Comment:
I think making this function external and storing the array in calldata would make the withdraw transaction cheaper.

Reply:
I can surely make both of these suggestions. In fact I meant to use `calldata`, but it got changed while fighting with failing testcases.

Reply:
Yes, `external` is better here IMO since we don't ever intend on calling this internally.